spec/factories/course_condition_levels.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :course_condition_level, class: Course::Condition::Level.name do
+    description "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.